src/layer/boundary.js

```diff
@@ -3,304 +3,289 @@
     ["==", ["get", "maritime"], 0],
   ],
   minzoom: 2,
-  layout: {
-    "line-cap": "round",
-    "line-join": "round",
-    visibility: "visible",
-  },
+  layout: boundaryLayout,
   source: "openmaptiles",
   "source-layer": "boundary",
 };
 
-export const country = {
-  id: "boundary_country",
+export const administrative = {
+  id: "boundary_administrative",
   type: "line",
   paint: {
-    "line-color": {
-      base: 1.2,
-      stops: [
-        [3, `hsl(${Color.hueBorder}, 2%, 47%)`],
-        [7, `hsl(${Color.hueBorder}, 2%, 37%)`],
+    "line-color": [
+      "step",
+      ["zoom"],
+      [
+        "match",
+        ["get", "admin_level"],
+        2,
+        `hsl(${Color.hueBorder}, 2%, 47%)`,
+        3,
+        `hsl(${Color.hueBorder}, 2%, 47%)`,
+        4,
+        `hsl(${Color.hueBorder}, 2%, 60%)`,
+        /* else */ Color.border,
       ],
-    },
-    "line-opacity": {
-      base: 1,
-      stops: [
-        [0, 0.4],
-        [4, 1],
-        [5, 0.8],
+      5,
+      [
+        "match",
+        ["get", "admin_level"],
+        2,
+        `hsl(${Color.hueBorder}, 2%, 42.2%)`,
+        3,
+        `hsl(${Color.hueBorder}, 2%, 47%)`,
+        4,
+        `hsl(${Color.hueBorder}, 2%, 54.3%)`,
+        /* else */ Color.border,
       ],
-    },
-    "line-width": {
-      stops: [
-        [2, 1],
-        [4, 1.5],
-        [10, 2.25],
+    ],
+    "line-dasharray": [
+      "step",
+      ["zoom"],
+      ["literal", [1000, 0]],
+      3,
+      [
+        "match",
+        ["get", "admin_level"],
+        2,
+        ["literal", [10, 2, 2, 2]],
+        3,
+        ["literal", [3, 3, 3, 3, 20, 3]],
+        4,
+        ["literal", [3, 3, 3, 3, 20, 3]],
+        5,
+        ["literal", [5, 4]],
+        6,
+        ["literal", [3, 3]],
+        /* else */ ["literal", [2, 4]],
       ],
-    },
-    "line-dasharray": {
-      stops: [[3, [10, 1, 3, 1]]],
-    },
-    "line-blur": 0,
+    ],
+    "line-opacity": [
+      "step",
+      ["zoom"],
+      ["match", ["get", "admin_level"], 2, 0.65, /* else */ 0],
+      3,
+      ["match", ["get", "admin_level"], 2, 0.85, 3, 1, 4, 1, /* else */ 0],
+      4,
+      ["match", ["get", "admin_level"], 2, 1, 3, 1, 4, 1, /* else */ 0],
+      5,
+      ["match", ["get", "admin_level"], 2, 0.8, 3, 1, 4, 1, /* else */ 0],
+      6,
+      [
+        "match",
+        ["get", "admin_level"],
+        2,
+        0.8,
+        3,
+        1,
+        4,
+        1,
+        5,
+        1,
+        6,
+        1,
+        /* else */ 0,
+      ],
+      11,
+      1,
+    ],
+    "line-width": [
+      "step",
+      ["zoom"],
+      ["match", ["get", "admin_level"], 2, 1, 3, 0.5, 4, 0.5, /* else */ 1],
+      4,
+      ["match", ["get", "admin_level"], 2, 1.5, 3, 0.5, 4, 0.5, /* else */ 1],
+      5,
+      ["match", ["get", "admin_level"], 2, 1.5, 3, 0.8, 4, 0.8, /* else */ 1],
+      7,
+      ["match", ["get", "admin_level"], 2, 2.115, 3, 1.1, 4, 1.1, /* else */ 1],
```

Comment:
At some zoom levels like z11, the line is thinner and fainter than it used to be.

Before | After
----|----
[<img width="400" height="300" alt="Thicker" src="https://github.com/user-attachments/assets/8a065650-270c-493e-b1e4-30e994eef70a" />](https://americanamap.org/#map=11/40.6005/-93.9365) | [<img width="400" height="300" alt="Thinner" src="https://github.com/user-attachments/assets/cd0f0613-3c39-4acd-bcf0-06fb6c34d684" />](https://preview.americanamap.org/pr/1093/#map=11/40.6005/-93.9365)